src/redux/Store.js

```diff
@@ -1,12 +1,12 @@
 /* eslint-disable import/no-extraneous-dependencies */
```

Comment:
- [ ] Kindly remove linter disablings and fix the resulting linter error manually.